### ISSUE TITLE
TokenManager: Only one token can be saved when Cookie is used as storage

### ISSUE THREAD

**Issue by arminhaghi** [okta/okta-auth-js#370]
When `storage` is set to `cookie` for `tokenManager` in config.

Token manager will only save one token on implicit flow or when used specifically.

Below code results in `idToken` being saved as the cookie value, and if order of the lines were swapped only `accessToken` will be saved as the cookie value. However, since the cookie value is json, it can be expanded to save both tokens as its value.

```
authClient.tokenManager.add("idToken", tokensResponse.tokens.idToken);
authClient.tokenManager.add("accessToken", tokensResponse.tokens.accessToken);
```

Example cookie value
```
{
    "idToken": {
        "value": "XXX",
        "idToken": "XXX",
        "claims": {
            ...
        },
        "expiresAt": 1588823020,
        "scopes": ["openid"],
        ...
    }
}
```

**Comment by swiftone** [comment]
Internal ref: OKTA-296918

**Comment by swiftone** [comment]
@arminhaghi - Thanks for the clear report, that definitely looks like a bug.

**Comment by shuowu** [comment]
@arminhaghi The fix is released in v3.1.4. Also, you can find the fix PR here: #402
